playbook/app/pb_kits/playbook/pb_dropdown/_dropdown.tsx

```diff
@@ -36,6 +36,7 @@ type DropdownProps = {
     onSelect?: (arg: GenericObject) => null;
     options: GenericObject;
     triggerRef?: any;
+    variant?: string;
```

Comment:
@nickamantia Shouldn't this variant be "primary" or "subtle" like Rails? We would also have to set the default to primary.

Reply:
great catch! thanks Carlos!